app/templates/AboutPage.html

```diff
@@ -52,21 +59,87 @@ <h1>Why Choose Career Compass?</h1>
                             <li><strong>Ethical Practices:</strong> We adhere to the highest standards of integrity, fairness, and confidentiality in all our interactions, ensuring a positive experience for both job seekers and employers.</li>
                         </ul>
                     </p>
-                    
-                    <!-- Our commitment to Diversity and Inclusion -->
-                    <h1>Our Commitment to Diversity and Inclusion</h1>
-                    <p>
-                        At Career Compass, we believe that diversity is not only a moral imperative but also a strategic advantage. We are committed to fostering an inclusive environment where individuals from all backgrounds feel valued, respected, and empowered to thrive. Through our recruitment practices and partnerships, we strive to promote diversity, equity, and inclusion in the workplace.
-                    </p>
-                    
-                    <!-- Get started with Career Compass today! -->
-                    <h1>Get Started with Career Compass Today!</h1>
-                    <p>
-                        Whether you're a job seeker embarking on a new career journey or an employer seeking top talent, Career Compass is here to help you navigate your way to success. Join our community today and let us guide you toward a brighter future!
+                </div>
+            </div>
+        </div>
+
+        <div class = "row">
+            <div class="col-md-12">
+                <div class="feedBox feedBoxSwell" id = "DivStarted">
+                    <h1 class="d-flex justify-content-between align-items-center">
+                        <!-- feed box title-->
+                        <span id = 'DivStartedTitle'>Our Commitment to Diversity and Inclusion</span>
+                        <span>
+                            <!-- feedbox button next to feed box title-->
+                            <button id = DivStartedButton class = 'btn feed_post_btn btnSwell'><i class="fa-solid fa-less-than"></i></button>
+                        </span>
+                    </h1>
+
+                    <p id = DivStartedContent>At Career Compass, we believe that diversity is not only a moral imperative but also a strategic advantage. We are committed 
+                        to fostering an inclusive environment where individuals from all backgrounds feel valued, respected, and empowered to thrive. Through our recruitment 
+                        practices and partnerships, we strive to promote diversity, equity, and inclusion in the workplace.
                     </p>
                 </div>
             </div>
         </div>
     </div>
+
+<script>
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Nice use of event listeners of domcontentloaded to ensure document is parsed before loading in content.